Formula/o/ocicl.rb

```diff
@@ -0,0 +1,46 @@
+class Ocicl < Formula
+  desc "OCI-based ASDF system distribution and management tool for Common Lisp"
+  homepage "https://github.com/ocicl/ocicl"
+  url "https://github.com/ocicl/ocicl/archive/refs/tags/v2.3.4.tar.gz"
+  sha256 "be31c1d5de3352bf2ddc759153abc9c04ce4c0b962cc1d9d07888379e18ff202"
+  license "MIT"
+
+  depends_on "sbcl" => [:build, :test]
+  depends_on "zstd"
+  depends_on "oras"
+
+  def install
+    system "sbcl", "--dynamic-space-size", "3072", "--no-userinit", "--eval",
+           "(require 'asdf)", "--eval",
+           "(progn (push (uiop:getcwd) asdf:*central-registry*) (asdf:make :ocicl) (sb-ext:quit))"
```

Comment:
```suggestion
    system "sbcl", "--dynamic-space-size", "3072", "--no-userinit", "--eval", <<~LISP
      (require 'asdf)
      (progn (push (uiop:getcwd) asdf:*central-registry*) (asdf:make :ocicl) (sb-ext:quit))
    LISP
```
Wouldn't this be easier? Not sure if the `'asdf` is missing a second `'`.